index.js

```diff
@@ -47,7 +47,7 @@ function addEventListenersByView(st) {
     );
 
   // Add event listeners for the Form view
```

Comment:
You removed the code for the 'form' in most places throughout the SPA, so this code for the form needs to be removed as well no?